edgedns/client.go

```diff
@@ -0,0 +1,110 @@
+package edgedns
+
+import (
+	"net/http"
+	"net/url"
+
+	"github.com/corbaltcode/go-akamai"
+	"github.com/corbaltcode/go-akamai/internal/request"
+)
+
+const defaultTTL = 300
+
+const (
```

Comment:
Would it make sense to make a `RecordType` enum (`type RecordType string` and consts for known values)?